src/components/header/style.module.css

```diff
@@ -381,7 +381,7 @@
 
 	svg {
 		width: 1.5rem;
-		height: auto;
+		height: 1.5rem;
```

Comment:
The Twitter icon is ever so slightly taller than the others, this tightens it up so that it's perfectly inline.